main.nf

```diff
@@ -966,7 +966,7 @@ workflow.onComplete {
     }
 
     // Switch the embedded MIME images with base64 encoded src
-    ngichipseqlogo = new File("$baseDir/assets/nf-core/ChIPseq_logo.png").bytes.encodeBase64().toString()
+    ngichipseqlogo = new File("$baseDir/assets/NGI-ChIPseq_logo.png").bytes.encodeBase64().toString()
```

Comment:
Instead of changing the file path, it would be better to rename the image itself so that it's updated with the rest of the pipeline.

Reply:
There was no "nf-core" path actually in the assets folder. That's why it was broke.

Reply:
Yes, I realise this. But `NGI-ChIPseq` is the old name for the pipeline, so it makes sense to update the image filename instead of just changing the reference to it in the code. The path above is a mistake from a bulk find and replace I did to change the name.

So I'm suggesting that it would be best to rename the image file to something like `nf-core-chipseq-logo.png` and then update the above path to match that new filename.

Reply:
Ah wait, of course the image itself also needs updating as it's the old logo. Ok, so we need to totally replace this file with the new logo (resized to be 150px tall).

Reply:
OK! No problem! I was just fixing the pipeline. I was able to run it after changing the image and changing the memory requirements (which as you told me there was no need). I was running on a 32GB machine and it was blocking me the first time.